images/capi/packer/ami/packer.json

```
@@ -139,9 +139,19 @@
       "arch": "{{user `goss_arch`}}",
       "format": "{{user `goss_format`}}",
       "format_options": "{{user `goss_format_options`}}",
-      "tests": ["packer/goss/goss.yaml"],
+      "goss_file": "{{user `goss_entry_file`}}",
+      "inspect": "{{user `goss_inspect_mode`}}",
+      "tests": ["{{user `goss_tests_dir`}}"],
       "url": "{{user `goss_url`}}",
       "use_sudo": true,
+      "vars_inline": {
+        "kubernetes_source_type": "{{user `kubernetes_source_type`}}",
+        "kubernetes_cni_source_type": "{{user `kubernetes_cni_source_type`}}",
+        "containerd_version" : "{{user `containerd_version`}}",
+        "kubernetes_cni_version": "{{user `kubernetes_cni_semver_suffix`}}",
```

Rather than introducing a new variable to use here, I think you can use:
```suggestion
        "kubernetes_cni_version": "{{user `kubernetes_cni_semver` | replace "v" "" 1}}",
```